node/bundler.test.ts

```diff
@@ -359,14 +359,14 @@ test('Loads declarations and import maps from the deploy configuration', async (
 
   const manifestFile = await fs.readFile(resolve(distPath, 'manifest.json'), 'utf8')
   const manifest = JSON.parse(manifestFile)
-  const { bundles, routes } = manifest
+  const { bundles, function_config: functionConfig } = manifest
```

Comment:
Could this be just `config` rather than `function_config`?

Reply:
The idea behind calling it `function_config` is that it contains function-specific config, as opposed to "general" config that applies to the full bundle. I fear that `config` is a bit too generic. What do you think?